tests/installation/first_boot.pm

```diff
@@ -20,6 +20,7 @@ use utils qw(handle_login handle_emergency);
 use version_utils 'sle_version_at_least';
 
 sub run {
+    select_console('x11') if (check_var('BACKEND', 'ipmi'));
```

Comment:
why?

Reply:
In qa_net_boot_from_hdd.pm we change to the SOL console in order to control PXE boot, boot from HDD and boot the SUT.

- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/boot/qa_net_boot_from_hdd.pm#L19

In first boot in IPMI it is still on SOL console, so here I change to x11 console, that with IPMI backend seems to do what we expect, connecting to the VNC server in the SUT.

Reply:
Well, fine but please make the code state that. We would have a very hard time to find your comment on github explaining this. When I am asking "why"   I want to state that eventually others will have the same question so we should find a way that the same question can be answered also later. At best with self-explanatory code, next best a comment in code.